scripts/build-runtime.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
how about we call this `scripts/build-apis.sh` since we're not actually building the runtime (since it's not a binary and doesn't have any code generation bits like the APIs do.